src/gatsby-node.js

```diff
@@ -67,25 +69,34 @@ exports.onPreInit = async function(
 
     await ensureDir(cachePath)
 
-    const dirFiles = await readdir(dirPath)
-    const cacheFiles = await readdir(cachePath)
+    let dirFileCount
+    let cacheFileCount
+    if (verbose) {
+      dirFileCount = await readFileCount(dirPath)
+      cacheFileCount = await readFileCount(cachePath)
+    } else {
+      const dirFiles = await readdir(dirPath)
+      const cacheFiles = await readdir(cachePath)
+      dirFileCount = dirFiles.length
+      cacheFileCount = cacheFiles.length
+    }
 
-    console.log(
+    reporter.info(
       `plugin-netlify-cache: Restoring ${
-        cacheFiles.length
+        cacheFileCount
       } cached files for ${humanName} directory with ${
-        dirFiles.length
+        dirFileCount
```

Comment:
Actually after testing this code, since each netlify build is completely fresh, I'm not sure we need to log out already existing files, it'll always be 0?

Reply:
If you additionally cache files within `node_modules/.cache`, it might happen that netlify already restored files for you.

Still, that post build information would be pretty useful. Happy to merge code that implements it :)

Reply:
Ah true, I didn't think of the caches in `node_modules`. Will work on that post build info when I can! :)